billing-service/Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM gradle:6 as builder
+FROM gradle:6.9.0 as builder
 WORKDIR /usr/src/app
 COPY src ./src
 COPY build.gradle .
 RUN ["gradle", "bootJar"]
 
 EXPOSE 8080
 
-FROM openjdk:11
+FROM openjdk:8-alpine
```

Comment:
Are we sure billing wants to use JDK 8?
Maybe they left it as JDK 11 because they rely on JDK 11 specific features?

Reply:
okay should i change it for eveyone?